nncf/utils.py

```diff
@@ -349,7 +349,7 @@ def objwalk(obj, unary_predicate: Callable[[Any], bool], apply_fn: Callable, mem
     return obj
 
 
-def should_consider_scope(scope_str: str, target_scopes: List[str], ignored_scopes: List[str]):
+def pt_should_consider_scope(scope_str: str, target_scopes: List[str], ignored_scopes: List[str]):
```

Comment:
And why some function have prefix pt_ / tf_  ( like `should_consider_scope`) antoher have no (`get_filter_num`).
In my mind prefix is not need, import from specific file is enough.

Reply:
Removed prefix